main.mk

```diff
@@ -89,7 +88,15 @@ CFLAGS_DEFS+=-DBB_THREADED -D_REENTRANT -D_THREAD_SAFE
 CFLAGS_DEFS+=-DPER_THREAD_MALLOC -DMONITOR_STACK
 #CFLAGS_DEFS+=-DUSE_SYS_ALLOC
 
-CPPFLAGS+=$(CFLAGS_ARCHFLAGS) $(CFLAGS_DEFS) $(CFLAGS_INC)
+COMDB2_DEBUG=1
```

Comment:
This is a temporary addition in a separate commit to enable testing of debug builds by defaults.